projects/igniteui-angular/src/lib/core/i18n/grid-resources.ts

```diff
@@ -100,6 +100,7 @@ export interface IGridResourceStrings {
     igx_grid_snackbar_addrow_actiontext?: string;
     igx_grid_actions_edit_label?: string;
     igx_grid_actions_add_label?: string;
+    igx_grid_actions_add_child_label?: string;
```

Comment:
this will require another update to the i18n repo

Reply:
@ChronosSF The existing PR includes this string as well: https://github.com/IgniteUI/igniteui-angular-i18n/pull/42/files#diff-825bea74263aba1da60756f31aa7f819R106